test/blackbox-tests/test-cases/dune-cache/cache-disabled.t

```diff
@@ -0,0 +1,130 @@
+Set up project and cache directory
+
+  $ echo "(lang dune 3.20)" > dune-project
+ 
+  $ cat > config <<EOF
+  > (lang dune 3.20)
+  > (cache enabled)
+  > EOF
+
+  $ cat > dune << EOF
+  > (library
+  >  (name foo))
+  > EOF
+
+  $ cat > foo.ml << EOF
+  > let f x y = x + y
+  > EOF
+
+  $ export DUNE_CACHE_ROOT=$(pwd)/dune_test_cache
+  $ mkdir $DUNE_CACHE_ROOT
+
+Initial build, populating cache
+
+  $ dune build --config-file config
+  $ wc -l _build/log 
+  31 _build/log
+  $ ls $DUNE_CACHE_ROOT
+  files
+  meta
+  temp
+  values
+
+Second build, no-op as cache is warm
+
+  $ dune clean --config-file config
+  $ dune build --config-file config
+  $ wc -l _build/log 
+  17 _build/log
+
+Build with --cache=disabled, which should be a no-op as _build is already populated
+
+  $ dune build --cache=disabled --config-file config
+  $ wc -l _build/log 
+  22 _build/log
```

Comment:
given that _build is still populated from the previous build, why do we have more stuff being built?